test/metabase/test/data.clj

```diff
@@ -203,7 +204,7 @@
   driver's implementation of [[ddl.i/format-name]]. (Most databases use the default implementation of `identity`; H2
   uses [[clojure.string/upper-case]].) This function DOES NOT quote the identifier."
   [a-name :- FormattableName]
-  (ddl.i/format-name (tx/driver) (name a-name)))
+  (ddl.i/format-name (tx/driver) (u/->snake_case_en (name a-name))))
```

Comment:
Unrelated, but I added this so switching between the `lib.tu.macros` and `mt` versions of stuff like `id` or `mbql-query` doesn't require you to switch between `snake_case` and `kebab-case`